### ISSUE TITLE
What is the Difference Between a Batch and an Epoch in a Neural Network?

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#5643]
What is the Difference Between a Batch and an Epoch in a Neural Network?<br>
Stochastic gradient descent is a learning algorithm that has a number of hyperparameters. Two hyperparameters that often confuse beginners are the batch size and number of epochs. They are both integer values and seem to do the same thing.<br>
https://ift.tt/2LqTCS4
